src/frontend/src/realms.tsx

```diff
@@ -266,7 +266,7 @@ export const RealmForm = ({ existingName }: { existingName?: string }) => {
                             timer = setTimeout(async () => {
                                 // @ts-ignore
                                 realm.controllers = await userNameToIds(
-                                    input.split(","),
+                                    input.split(/[,\s]/),
                                 );
                                 setRealm({ ...realm });
                             }, 1500);
```

Comment:
You think 1.5s is too low to hit the next letter on mobile? Are you sure?

Reply:
Ideally we should have a dropdown of course...

Reply:
Yeah I guess so, although could be cumbersome as the list of users grows, unless it's a search/dropdown box. I miss the feature that got removed recently where recognised names are displayed beneath the input field (then the unrecognised names could get purged from the input field on submission, or when the field loses focus, rather than on a timer). Anyway, no major issue, just seemed worth commenting on.